install.sh

```diff
@@ -59,13 +61,42 @@ function install_apt {
     fi
 }
 
+function install_yum {
+    local CMD=""
+    if [ -n "${YUM_DNF}" ]; then
+        CMD=$YUM_DNF
+    else
+        if [ -n "${YUM_YUM}" ]; then
+            CMD=$YUM_YUM
+	fi
+    fi
+
+    if [ "${CMD}" != "" ]; then
+        local PARAMS="--assumeyes"
```

Comment:
The `yum`/`dnf` equivalent of this option is written without a hyphen in the middle. Although I just looked at the manpage at it seems `-y` can be used, too.